pkgs/development/compilers/coreclr/default.nix

```diff
@@ -73,7 +61,9 @@ stdenv.mkDerivation rec {
   ];
 
   buildPhase = ''
+    # set -x
     ./build.sh $BuildArch $BuildType
+    # set +x
```

Comment:
Did you left this unintentionally?

Reply:
Yep, was troubleshooting. Will be removed in the final version. Thanks, @Mic92